### DISCUSSION TITLE
Conda recipe

### DISCUSSION THREAD

**Discussion by spenczar** [ParallelSSH/parallel-ssh#306] [General]
Hi,

I'd like to use this (really excellent) package in some software which is typically installed with Conda. It would be nice to have a recipe uploaded to conda-forge for this purpose.

This is easy enough for me to do myself with [grayskull](https://github.com/conda-incubator/grayskull), but I wanted to check here first, since if I do it myself then I'll be marked as the maintainer of the recipe, which doesn't quite seem right.

**Comment by pkittenis** [comment]
Hello,

Thanks for the interest. I had considered making conda recipes in the past but do not have the bandwidth to support them, plus it is duplicated effort that has already gone into publishing binary wheels.

If others want to make one I'm happy for them to do it and support it.

Note - Making a conda recipe for this package is straight forward since it is purely python code, for its two dependencies, ssh-python and ssh2-python less so as they are native code extensions.

It is that part that was found too time consuming, especially considering the work gone into making binary wheels for them for all platforms.
